### DISCUSSION TITLE
Meta Icon Fails in Safari

### DISCUSSION THREAD

**Discussion by mtanco** [h2oai/wave#861] [Q&A] [answered]
The meta icon example works well in Chrome but is ignored by Safari which just shows the standard Wave icon for the below example:

https://github.com/h2oai/wave/blob/master/py/examples/meta_icon.py

**Comment by lo5** [comment, marked as answer]
We couldn't figure out how to get it to work in Safari - the API docs mention this: https://wave.h2o.ai/docs/api/ui#meta_card

**Reply by lo5** [reply]
Related: https://github.com/h2oai/wave/issues/548

**Reply by mtanco** [reply]
Right in the issues already, sorry for the dup!
